tools.go

```diff
@@ -46,6 +46,8 @@ func main() {
 		baseLocalDir = os.Getenv("APPDATA")
 	case "linux":
 		baseLocalDir = filepath.Join(os.Getenv("HOME"), ".local", "share")
+	case "darwin":
+		baseLocalDir = filepath.Join(os.Getenv("HOME"), ".ficsit-cli")		
```

Comment:
Same thing as for the other `baseLocalDir`

Reply:
done